cmd/neofs-adm/internal/modules/root.go

```diff
@@ -0,0 +1,57 @@
+package modules
+
+import (
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-adm/internal/modules/config"
+	"github.com/nspcc-dev/neofs-node/misc"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	rootCmd = &cobra.Command{
+		Use:   "neofs-adm",
+		Short: "NeoFS Administrative Tool",
+		Long: `NeoFS Administrative Tool provides functions to setup and
+manage NeoFS network deployment.`,
+		RunE:         entryPoint,
+		SilenceUsage: true,
+	}
+
+	configFlag = "config"
+)
+
+func init() {
+	cobra.OnInitialize(func() { initConfig(rootCmd) })
+	// we need to init viper config to bind viper and cobra configurations for
+	// rpc endpoint, alphabet wallet dir, key credentials, etc.
+
+	rootCmd.PersistentFlags().StringP(configFlag, "c", "", "config file")
+	rootCmd.Flags().Bool("version", false, "application version")
+
+	rootCmd.AddCommand(config.RootCmd)
+}
+
+func Execute() error {
+	return rootCmd.Execute()
+}
+
+func entryPoint(cmd *cobra.Command, args []string) error {
+	printVersion, err := cmd.Flags().GetBool("version")
+	if err == nil && printVersion {
+		cmd.Printf("neofs-adm %s\n", misc.Version)
```

Comment:
in https://github.com/nspcc-dev/neofs-node/commit/eadc3a4de94cfdfd621bdf61daeccdcfbd4505c1 i made all "version outputs" be in the same format, wouldn't it be better to make the same here? or to change the format in `cli`/`IR`/`SN`